src/common/libjob/job.h

```diff
@@ -107,11 +107,11 @@ flux_future_t *flux_job_list (flux_t *h,
                               int states);
 
 /* Similar to flux_job_list(), but retrieve inactive jobs newer
- * than a timestamp.
+ * than since.
```

Comment:
I would just drop this change because the old description made slightly more sense. Or, change to `...retrieve inactive jobs newer than the timestamp "since"`